float64 Time
string Limit
float64 Current_Az
float64 Current_El
float64 Command_Az
float64 Command_El
float64 Deviation_Az
float64 Deviation_El
string Drive_ready_Az
string Drive_ready_El
string Authority
string Emergency
float64 Current_Dome
string Door_Dome
string Door_Membrane
string Door_Authority
string Current_M4
string Current_Hot
float64 Year
float64 Month
float64 Day
float64 Hour
float64 Min
float64 Sec
float64 InTemp
float64 OutTemp
float64 InHumi
float64 OutHumi
float64 WindDir
float64 WindSp
float64 Press
float64 Rain
float64 CabinTemp1
float64 CabinTemp2
float64 DomeTemp1
float64 DomeTemp2
float64 GenTemp1
float64 GenTemp2
float64 Current_M2
float64 MJD
int64 LST
float64 Secofday
string from_node
float64 timestamp